lib/failed_autoredeems.ts

```diff
@@ -116,30 +116,27 @@ const isMatchingSender = async (
   l1Report: L1TicketReport | undefined
 ): Promise<boolean> => {
   if (deposit !== undefined) {
-    const depositSenderFromGraph = deposit.sender
+    const depositSenderFromGraph = deposit.sender.toLowerCase()
     const rec = await getL1TXRec(deposit.transactionHash)
-    const depositSenderFromRec = rec.from
-    if (depositSenderFromGraph === process.env.SENDER_ADDRESS) {
-      return true
-    }
-    if (depositSenderFromRec === process.env.SENDER_ADDRESS) {
+    const depositSenderFromRec = rec.from.toLowerCase()
+
+    if (
+      depositSenderFromGraph == process.env.SENDER_ADDRESS!.toLowerCase() ||
+      depositSenderFromRec == process.env.SENDER_ADDRESS!.toLowerCase()
+    ) {
       return true
-    } else {
-      return false
     }
   }
 
   if (l1Report !== undefined) {
-    const retryableSenderFromGraph = l1Report.sender
+    const retryableSenderFromGraph = l1Report.sender.toLowerCase()
     const rec = await getL1TXRec(l1Report.transactionHash)
-    const retryableSenderFromRec = rec.from
-    if (retryableSenderFromGraph === process.env.SENDER_ADDRESS) {
-      return true
-    }
-    if (retryableSenderFromRec === process.env.SENDER_ADDRESS) {
+    const retryableSenderFromRec = rec.from.toLowerCase()
+    if (
+      retryableSenderFromGraph === process.env.SENDER_ADDRESS!.toLowerCase() ||
+      retryableSenderFromRec === process.env.SENDER_ADDRESS!.toLowerCase()
+    ) {
```

Comment:
Seems like this block is pretty duplicative to the one above. Would be nice to refactor this check out into a function if we could then just call it on `deposit` and `l1Report`. Looks like we could have a common interface between `TokenDepositData` and `L1TicketReport` maybe